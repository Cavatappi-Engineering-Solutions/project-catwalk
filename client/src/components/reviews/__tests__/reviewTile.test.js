/* eslint-disable no-undef */
import React from 'react'
import renderer from 'react-test-renderer'
import Review from '../reviewTile.jsx'

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Review
        data={{
          rating: 5,
          summary: 'summary',
          body: 'body',
          recommend: true,
          response: 'response',
          helpfulness: 4,
          review_id: 1234,
          date: '2021-06-19T00:00:00.000Z'
        }}
      />
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
