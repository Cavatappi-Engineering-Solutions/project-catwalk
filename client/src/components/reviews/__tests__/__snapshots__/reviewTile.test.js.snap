// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="review-tile"
>
  <div
    className="tile-header"
  >
    <div
      className="star-rating"
      style={
        Object {
          "--rating": "100%",
        }
      }
    >
      ★★★★★
    </div>
    <span
      className="tile-date"
    >
      undefined, undefined
    </span>
  </div>
  <p
    className="tile-summary"
  >
    summary
  </p>
  <p
    className="tile-body"
  >
    body
  </p>
  <p
    className="tile-recommend"
  />
  <p
    className="tile-response"
  >
    response
  </p>
  <div
    className="tile-helpful-report"
  >
    <span
      className="tile-helpful"
    >
      Helpful?
    </span>
    <span
      className="tile-yes"
      id="yes-1234"
      onClick={[Function]}
    >
      Yes
    </span>
    <span
      className="tile-helpfulness"
    >
      (
      4
      )
    </span>
    <span
      className="tile-report"
    >
      Report
    </span>
  </div>
</div>
`;
